packages/flutter/lib/src/services/raw_keyboard.dart

```diff
@@ -604,8 +609,38 @@ class RawKeyboard {
     _listeners.remove(listener);
   }
 
-  /// A handler for hardware keyboard events that will stop propagation if the
-  /// handler returns true.
+  final List<RawKeyEventHandler> _keyHandlers = <RawKeyEventHandler>[];
+
+  /// Register a handler that is called when the user presses or releases
+  /// a hardware keyboard key if the [keyEventHandler] didn't handle it.
+  ///
+  /// These events will be delivered after the [keyEventHandler] (typically set
```

Comment:
Do we still need `keyEventHandler` then? Or could the focus system also just use this mechanism?